data/scripts/convert-config-to-json.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
If there's no reason to have this script in the tree, as stated in the previous version of this PR, just leave it out.